### PR TITLE
395759 update client for token endpoint

### PR DESCRIPTION
**Pull request by matt-yin-lf**
1. Updated the LfdsUsernamePassword handler to work with the `/Token` endpoint.
2. In the configuration (`.env`), removed `Organization`, added `GrantType`.
3. Renamed `Lfds` to 'SelfHosted'.

### CODE REVIEW

File: lfds-nswag.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
   "defaultVariables": null,
   "documentGenerator": {
     "fromDocument": {
-      "url": "lfds-swagger.json",
+      "url": "selfhosted-swagger.json",
```
Comment (jason-chan-lf) [inline lfds-nswag.json]: this file could also be renamed to `selfhosted-nswag.json` to be consistent

File: tests/integration/LfdsUsernamePasswordHandlerTests.cs [hunk 2]
```diff
@@ -7,28 +7,28 @@
...
+using Laserfiche.Api.Client.SelfHosted;
 using System.Net;
 
 namespace Laserfiche.Api.Client.IntegrationTest
 {
     [TestClass]
-    [TestCategory("LFDS")]
-    public class LfdsUsernamePasswordHandlerTest : BaseTest
+    [TestCategory("SelfHosted")]
+    public class SelfHostedUsernamePasswordHandlerTest : BaseTest
```
Comment (jason-chan-lf) [inline tests/integration/LfdsUsernamePasswordHandlerTests.cs]: could also rename the file to match the class name

File: selfhosted-nswag.json [hunk 3]
```diff
@@ -3,16 +3,16 @@
...
       "output": null,
       "newLineBehavior": "Auto"
     }
   },
   "codeGenerators": {
     "openApiToCSharpClient": {
       "classname": "{controller}Client",
-      "namespace": "Laserfiche.Api.Client.Lfds",
-      "output": "src/Lfds/AccessTokensClient.cs",
+      "namespace": "Laserfiche.Api.Client.SelfHosted",
```
Comment (jason-chan-lf) [inline selfhosted-nswag.json]: also need to update this file from SelfHosted -> APIServer

File: selfhosted-swagger.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
+{
```
Comment (jason-chan-lf) [inline selfhosted-swagger.json]: and i guess could also rename the file to apiserver-swagger.json

File: selfhosted-nswag.json (hunk 3 above)
Comment (matt-yin-lf) [reply selfhosted-nswag.json]: done

File: selfhosted-swagger.json (hunk 4 above)
Comment (matt-yin-lf) [reply selfhosted-swagger.json]: done

**Review by jason-chan-lf — APPROVED** [review]

**Pull request merged**